test/cli-build.spec.js

```diff
@@ -135,6 +137,7 @@ describe('cli build', () => {
           expect(removeSpy).toHaveBeenCalledWith(
             skyPagesConfigUtil.spaPathTemp()
           );
+          expect(passedConfig.skyuxPathAlias).toBeUndefined();
 
```

Comment:
Should we also have a test that checks for the presence of `import { SkyModule } from 'whatever'` in the resulting module file?  That way we don't have to worry about how we actually implemented overriding default behavior because we're ensuring the output itself is correct.